pkgs/tools/admin/azure-cli/python-packages.nix

```diff
@@ -6,7 +6,7 @@ let
     prePatch = (attrs.prePatch or "") + ''
       rm -f azure_bdist_wheel.py tox.ini
       substituteInPlace setup.py \
-        --replace "cryptography>=2.3.1,<3.0.0" "cryptography"
+        --replace "cryptography>=3.2,<3.4" "cryptography"
```

Comment:
updated this substituteInPlace https://hydra.nixos.org/build/146088369/nixlog/99
`substituteStream(): WARNING: pattern 'cryptography>=2.3.1,<3.0.0' doesn't match anything in file 'setup.py'`